### PR TITLE
document head element needed for self-injection

### LINKED ISSUE #37: Injection into head element is not documented
I discovered from trial, error, and research that Capacitor will attempt to inject itself into the `head` element of the HTML page, as discussed in the following places:

- https://github.com/ionic-team/capacitor/issues/1572#issuecomment-495720601
- https://github.com/ionic-team/capacitor/issues/1572#issuecomment-495726425
- https://forum.ionicframework.com/t/unable-to-inject-capacitor/178713

I encountered this behavior when migrating a small test app from Cordova to Capacitor.

I had originally proposed an update in <https://github.com/ionic-team/capacitor/pull/3024>, will recreate it in this repo. Sure beats the manual injection requirement of Cordova.

### PR DESCRIPTION
**Pull request by brodybits**
resolves #37

as originally proposed in <https://github.com/ionic-team/capacitor/pull/3024>

I have a feeling it should be possible to make this flow better, wouldn't mind ideas or suggestions for rework.

### CODE REVIEW

File: docs-md/getting-started/index.md [hunk 1]
```diff
@@ -27,7 +27,7 @@ make sure you update CocoaPods using `pod repo update` before starting a new pro
 
 ## Adding Capacitor to an existing web app
 
-Capacitor was designed to drop into any existing modern JavaScript web app. A valid `package.json` file and a folder containing all web assets are required to get started.
+Capacitor was designed to drop into any existing modern JavaScript web app. A valid `package.json` file and a folder containing all web assets are required to get started. Capacitor assumes that it can inject itself into the `head` element in the main `index.html` file.
```
Comment (dotNetkow) [inline docs-md/getting-started/index.md:30]:
```suggestion
Capacitor was designed to drop into any existing modern JavaScript web app. A valid `package.json` file and a folder containing all web assets are required to get started. Additionally, ensure that a `<head>` element is available in the main `index.html` file, as Capacitor is injected there upon app initialization.
```

**Comment by dotNetkow** [comment]
Thanks Chris for documenting this! I reworded the sentence just a bit.

**Comment by brodybits** [comment]
> I reworded the sentence just a bit.

I don't see it

File: docs-md/getting-started/index.md [hunk 2]
```diff
@@ -27,7 +27,7 @@ make sure you update CocoaPods using `pod repo update` before starting a new pro
 
 ## Adding Capacitor to an existing web app
 
-Capacitor was designed to drop into any existing modern JavaScript web app. A valid `package.json` file and a folder containing all web assets are required to get started.
+Capacitor was designed to drop into any existing modern JavaScript web app. A valid `package.json` file and a folder containing all web assets are required to get started. Additionally, ensure that a `<head>` element is available in the main `index.html` file, as Capacitor is injected there upon app initialization.
```
Comment (brodybits) [inline docs-md/getting-started/index.md:30]: Edit accepted, thanks. I am assuming we would show up as co-authors in the merge. I did take the liberty to update the title as well.

Another idea, wonder if it makes sense or not:

```suggestion
Capacitor was designed to drop into any existing modern JavaScript web app. A valid `package.json` file and a folder containing all web assets are required to get started. In addition, a `<head>` element is needed in the main `index.html` file, as Capacitor is injected there upon app initialization.
```

**Comment by dotNetkow** [comment]
All the credit to you! Thanks again :)

**Comment by brodybits** [comment]
Thanks for your help with this, yeah!

**Pull request merged**